src/DGtal/shapes/parametric/Ball3D.ih

```diff
@@ -83,7 +84,14 @@ DGtal::Ball3D<T>::parameter( const RealPoint & pp ) const
   AngularCoordinates angle;
   
   angle.first = atan2( p[1], p[0] );
-  angle.second = acos( sqrt(p[2] * p[2]) / p.norm() );
+  double tmp = abs( p[2] );
```

Comment:
double tmp = std::min( abs( p[2] ), 1.0 );
double pn = p.norm(); // otherwise computed twice
angle.second = ( pn > 0.0 ) ? acos( tmp / pn ) : 0.0;

Reply:
@JacquesOlivierLachaud Github as a new "suggestion" feature if you want (the file with + icon)
<img width="296" alt="capture d ecran 2018-11-04 a 22 46 41" src="https://user-images.githubusercontent.com/700165/47970269-8ca8a400-e083-11e8-8341-c8d8df5c1056.png">

Reply:
the PR author can "apply suggestion" easily (great for typos or minor stuff)